looks like UCR might have be depending on the changes to group by you made:

```

======================================================================
ERROR: corehq.apps.userreports.tests.test_report_aggregation:TestReportAggregationSQL.test_total_row
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_report_aggregation.py", line 382, in test_total_row
    view.export_table
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/reports/view.py", line 519, in export_table
    return self.report_export.get_table()
  File "/vendor/lib/python2.7/site-packages/memoized.py", line 13, in _inner
    return m(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/memoized.py", line 133, in __call__
    value = self.func( *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/reports/util.py", line 87, in get_table
    total_rows = [self.data_source.get_total_row()] if self.data_source.has_total_row else []
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/reports/data_source.py", line 129, in get_total_row
    return self.data_source.get_total_row()
  File "/vendor/lib/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/decorators.py", line 41, in _inner
    raise UserReportsError(six.text_type(e))
UserReportsError: Missing column in table (config_report_TEST_DOMAIN_foo_78856a50): u'sum_report_column_display_number'
----------------------------------------------------------------------
```

--- a/requirements/dev-requirements.txt
+++ b/requirements/dev-requirements.txt
@@ -193,7 +193,7 @@ snowballstemmer==1.2.1    # via sphinx
 socketpool==0.5.3
 sphinx-rtd-theme==0.1.9
 sphinx==1.3.1
-sqlagg==0.11.0
+sqlagg==0.11.1
 sqlalchemy-postgres-copy==0.5.0
 sqlalchemy==1.2.12
 sqlparse==0.2.4           # via django-debug-toolbar

--- a/requirements/docs-requirements.txt
+++ b/requirements/docs-requirements.txt
@@ -152,7 +152,7 @@ sphinx-rtd-theme==0.3.0
 sphinx==1.7.2
 sphinxcontrib-django==0.3.1
 sphinxcontrib-websupport==1.1.0  # via sphinx
-sqlagg==0.11.0
+sqlagg==0.11.1
 sqlalchemy==1.2.12
 stripe==1.67.0
 suds-jurko==0.6

--- a/requirements/prod-requirements.txt
+++ b/requirements/prod-requirements.txt
@@ -164,7 +164,7 @@ singledispatch==3.4.0.3   # via tornado
 six==1.11.0
 smartypants==2.0.1
 socketpool==0.5.3
-sqlagg==0.11.0
+sqlagg==0.11.1
 sqlalchemy==1.2.12
 stripe==1.67.0
 suds-jurko==0.6

--- a/requirements/requirements.in
+++ b/requirements/requirements.in
@@ -51,7 +51,7 @@ numpy==1.7.1
 six==1.11.0
 socketpool==0.5.3
 markdown==2.2.1
-sqlagg==0.11.0
+sqlagg==0.11.1
 django-redis==4.9.0
 redis==2.10.6
 redis-py-cluster==1.3.5

--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -142,7 +142,7 @@ simplejson==3.11.1
 six==1.11.0
 smartypants==2.0.1        # via pycco
 socketpool==0.5.3
-sqlagg==0.11.0
+sqlagg==0.11.1
 sqlalchemy==1.2.12
 stripe==1.67.0
 suds-jurko==0.6

--- a/requirements/test-requirements.txt
+++ b/requirements/test-requirements.txt
@@ -152,7 +152,7 @@ simplejson==3.11.1
 six==1.11.0
 smartypants==2.0.1
 socketpool==0.5.3
-sqlagg==0.11.0
+sqlagg==0.11.1
 sqlalchemy-postgres-copy==0.5.0
 sqlalchemy==1.2.12
 stripe==1.67.0

